torch/csrc/jit/runtime/register_ops_common_utils.cpp

```diff
@@ -0,0 +1,119 @@
+#include <torch/csrc/jit/runtime/register_ops_common_utils.h>
```

Comment:
I feel all the helper functions in this file are used by mobile side, so maybe it's better to move this and the header under `jit/mobile` folder?